Fixed router path to simplify deployment

draymond63 opened the pull request:
Filezilla needs a base url of "/", whereas github pages needs "/website/". To simplifying deployment from Github pages to the release build, the base url should dynamically change.

draymond63 committed:
Fixed router path to simplify deployment

namansood commented:
Looks good! When I did the deployment, I also changed the path in `vue.config.js`, though I'm not entirely sure if that was necessary. Could you confirm that it's not needed?

draymond63 committed:
Removed vue-config.js

draymond63 commented:
> Looks good! When I did the deployment, I also changed the path in `vue.config.js`, though I'm not entirely sure if that was necessary. Could you confirm that it's not needed?

Good point! I've updated the vue config to match the router's conditional.
